src/OnboardingSPA/steps/SiteGen/SocialMedia/stylesheet.scss

```diff
@@ -0,0 +1,104 @@
+.nfd-onboarding-step {
+
+	&--site-gen {
+
+		&__social-media {
+
+			&__container {
+
+				&__heading {
+					display: flex;
+					flex-direction: row;
+					align-items: center;
+					justify-content: center;
+
+					&__animation {
+						background-image: var(--sitegen-ai-animation);
+						background-repeat: no-repeat;
+						background-size: contain;
+						width: 83px;
+						height: 72px;
+					}
+
+					&__text {
+						color: var(--nfd-onboarding-primary);
+						font-size: clamp(0.875rem, 0.7727rem + 0.5114vw, 2rem);
+						margin-left: 15px;
+						white-space: normal;
+						line-height: 1;
+					}
+				}
+
+				&__buttons {
+					display: flex;
+					flex-direction: row;
+					align-items: center;
+					justify-content: flex-end;
+					padding: 16px;
+
+					&__skip {
+						font-size: clamp(0.5rem, 0.4091rem + 0.4545vw, 1.5rem);
+						font-style: normal;
+						font-weight: 510;
+						margin-right: 24px;
+
+						&:hover {
+							text-decoration: underline;
+							color: var(--nfd-onboarding-primary);
+						}
+					}
+				}
+
+			}
+
+			&__contain {
+				width: 600px;
+				margin: 16px;
+				border: 1.25px dashed var(--nfd-onboarding-site-logo-border);
```

Comment:
Should this be dotted?
<img width="1617" alt="Screenshot 2023-11-20 at 1 18 49 PM" src="https://github.com/newfold-labs/wp-module-onboarding/assets/48691514/4ed19d16-d156-4cc5-8285-fbe87b04dcb5">